### PR TITLE
Update french.txt

### PR DESCRIPTION
**Pull request by archimede67**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Comment by def-** [comment]
@Chairn want to take a look?

File: data/languages/french.txt [hunk 1]
```diff
@@ -1517,64 +1518,64 @@ Loading sound files
...
-== 
+== Initialisation de la logique de la map
 
 Sending initial client info
-== 
+== Envoi des informations initiales du client
 
 Uploading map data to GPU
-== 
+== Téléchargement des données de la map vers le GPU
```
Comment (Chairn) [inline data/languages/french.txt:1539]: "Transfert" is better. "Téléchargement" is literally remote loading, which is not the case here

Comment (archimede67) [reply data/languages/french.txt]: Yeah I agree

**Pull request merged**